sdk/core/core-sse/src/utils.ts

```diff
@@ -65,7 +62,7 @@ function iteratorToStream<T>(
 }
 
 export function ensureAsyncIterable(
-  stream: Nullable<IncomingMessage, "socket"> | ReadableStream<Uint8Array>,
+  stream: IncomingMessage | NodeJSReadableStream | ReadableStream<Uint8Array>,
```

Comment:
Do we still need `IncomingMessage`?

Reply:
Agreed, at runtime, it doesn't matter. From types point of view, it is a breaking change to remove it now.